docs/advanced/configure-cf-external-databases-using-azure-mysql-postgres-service/migrate-mysql-databases/README.md

```diff
@@ -0,0 +1,97 @@
+# Migrate Cloud Foundry MySQL databases to Azure Database for MySQL
+
+On Cloud Foundry (CF), it is recommended to use PAAS service for the purpose of high availability. [Azure Database for MySQL](https://azure.microsoft.com/en-us/services/mysql/) is one of services that CF can levege for HA. 
+
+For a new deployment, you can refer to this [guidance](../README.md) to use Azure Database for MySQL as external databases.
+For an existed deployment, if you want to use Azure Database for MySQL, this document will provide a simple guildance to migrate the database.
+
+**NOTE:** Be careful on operating databases, you should practise on a test environment before officially do the migration on your production environment.
```

Comment:
hmmm, think about removing this paragraph, look like a bit redundant.

Reply:
no harm to remind, will keep this paragraph. thanks.